docs/docs/configuration/animations.mdx

```diff
@@ -138,22 +173,21 @@ The default configuration is defined here: <a href="https://github.com/chartjs/C
 
 Direct property configuration overrides configuration of same property in a collection.
 
-These defaults can be overridden in `options.animation` or `dataset.animation` and `tooltip.animation`. These keys are also [Scriptable](../general/options.md#scriptable-options)
+To enable a custom collection, the default one of the same type must be disabled.
 
 :::note
 These default collections are overridden by most dataset controllers.
 :::
 
-### Default modes
+## Disabling animation
 
-| Mode | Option | Value | Description
-| -----| ------ | ----- | -----
-| `'active'` | duration | 400 | Override default duration to 400ms for hover animations
-| `'resize'` | duration | 0 | Override default duration to 0ms (= no animation) for resize
-| `'show'` | colors | `{ type: 'color', properties: ['borderColor', 'backgroundColor'], from: 'transparent' }` | Colors are faded in from transparent when dataset is shown using legend / [api](../developers/api.md#showdatasetIndex).
-| | visible | `{ type: 'boolean', duration: 0 }` | Dataset visiblity is immediately changed to true so the color transition from transparent is visible.
-| `'hide'` | colors | `{ type: 'color', properties: ['borderColor', 'backgroundColor'], to: 'transparent' }` | Colors are faded to transparent when dataset id hidden using legend / [api](../developers/api.md#hidedatasetIndex).
-| | visible | `{ type: 'boolean', easing: 'easeInExpo' }` | Visibility is changed to false very at late phase of animation
+To disable an animation configuration, the animation node must be set to `false`.
+
+```javascript
+chart.options.animation = false; // disables the whole animation
+chart.options.animation.active = false; // disables the animation for 'active' mode
```

Comment:
Not sure this is true, it only disables the overrides for that mode and falls back to defaults.

See https://codepen.io/kurkle/pen/mdVZNBY?editors=1010
Upper chart is "normal", with 5s duration, lower one has `active: false` additionally.,

Reply:
uhm... got it. In this case it's till not clear to me how to disable. Setting duration:0 ?

Reply:
Yes, setting duration to 0 makes the hover effects immediate, so you could say its disabled.

I'm probably not understanding the use case for this.

Reply:
> I'm probably not understanding the/any use case for this.

I think that's my fault because I was convinced to have seen a sample where to disable the animation (for tooltip) it was implemented setting the tooltip.animation = false.

What do you think if I change as following?

## Disabling animation

To disable an animation configuration, the animation `duration` must be set to `0`.

```javascript
chart.options.animation.duration = 0; // disables the whole animation
chart.options.animation.active.duration = 0; // disables the animation for 'active' mode
chart.options.animation.colors.duration = 0; // disables the collection of colors properties
```

Reply:
Sorry, maybe I'm getting crazy.
Using your sample, and setting `chart2.options.animation.duration = 0;`, the chart is animated anyway.
Instead, if I set `chart2.options.animation = false;` it's not animated at all.

Where am I wrong?

Reply:
Original was correct, expect for the `'active' mode`. But you can not really disable a mode, you can only change the behavior.

Reply:
> Original was correct, expect for the `'active' mode`. But you can not really disable a mode, you can only change the behavior

Therefore :

```javascript
chart.options.animation= false; // CORRECT
chart.options.animation.active = false; // WRONG
chart.options.animation.active.duration = 0; // CORRECT
chart.options.animation.colors = false; // CORRECT
chart.options.animation.borderWidth = false; // CORRECT
```